[2024-04-23T20:06:17.698+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-23T20:06:17.714+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_report.transform manual__2024-04-23T19:57:27.600119+00:00 [queued]>
[2024-04-23T20:06:17.718+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_report.transform manual__2024-04-23T19:57:27.600119+00:00 [queued]>
[2024-04-23T20:06:17.719+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-04-23T20:06:17.726+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): transform> on 2024-04-23 19:57:27.600119+00:00
[2024-04-23T20:06:17.730+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2895) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-23T20:06:17.732+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'monthly_report', 'transform', 'manual__2024-04-23T19:57:27.600119+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/homework2.py', '--cfg-path', '/tmp/tmpsv_nwsk7']
[2024-04-23T20:06:17.733+0000] {standard_task_runner.py:63} INFO - Started process 2897 to run task
[2024-04-23T20:06:17.733+0000] {standard_task_runner.py:91} INFO - Job 9: Subtask transform
[2024-04-23T20:06:17.760+0000] {task_command.py:426} INFO - Running <TaskInstance: monthly_report.transform manual__2024-04-23T19:57:27.600119+00:00 [running]> on host 96e40b232902
[2024-04-23T20:06:17.813+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_report' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2024-04-23T19:57:27.600119+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-23T19:57:27.600119+00:00'
[2024-04-23T20:06:17.815+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-23T20:06:17.815+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-23T20:06:17.816+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: transfrom() missing 1 required positional argument: 'date'
[2024-04-23T20:06:17.822+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=monthly_report, task_id=transform, execution_date=20240423T195727, start_date=20240423T200617, end_date=20240423T200617
[2024-04-23T20:06:17.829+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 9 for task transform (transfrom() missing 1 required positional argument: 'date'; 2897)
[2024-04-23T20:06:17.837+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-04-23T20:06:17.848+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-23T20:06:17.849+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
